modules/knative-serving-concurrent-autoscaling-requests.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * serverless/configuring-knative-serving-autoscaling.adoc
+
+[id="knative-serving-concurrent-autoscaling-requests_{context}"]
+= Configuring concurrent requests for Knative Serving autoscaling
+
+You can specify the number of concurrent requests that should be handled by each instance of an application (revision container) by adding the `target` annotation or the `containerConcurrency` field in the revision template.
+
+Here is an example of `target` and `containerConcurrency` being used in a revision template.
```

Comment:
Not critical, but just wondering if it is a realistic example to use both in the same configuration, when the above sentence says that you can use one OR the other.